static/gsApp/components/features/planFeature.tsx

```diff
@@ -105,6 +109,17 @@ function PlanFeature({subscription, features, organization, children}: Props) {
     plans = billingConfig.planList;
   }
 
+  // HACK: we want to remove `global-views` from getsentry and move it to flagpole,
+  // but since PlanFeature hooks into getsentry to determine which plan
+  // `global-views` is in, we need to hardcode it into the plans here
+  for (const plan of plans) {
+    if (isBizPlanFamily(plan) || plan.id.includes('mm2') || isAmEnterprisePlan(plan.id)) {
```

Comment:
i believe `isBizPlanFamily` should already cover the mm2 plans that have `global-views`, but `plan.id.includes('mm2')` would also include plans that shouldn't have access